tcf_website/views/review.py

```diff
@@ -123,3 +87,23 @@ def get_object(self):  # pylint: disable=arguments-differ
             raise PermissionDenied(
                 "You are not allowed to delete this review!")
         return obj
+
+
+@login_required
+def edit_review(request, review_id):
+    """Review modification view."""
+    review = get_object_or_404(Review, pk=review_id)
+    if review.user != request.user:
+        raise PermissionDenied('You are not allowed to edit this review!')
+
+    if request.method == 'POST':
```

Comment:
Does this set the `modified` field of the `Review` model automatically?

Btw any insights on using `default=timezone.now` vs `auto_now` in the model? I think `auto_now` will update it automatically on `Model.save()` but not sure about the former.
https://docs.djangoproject.com/en/dev/ref/models/fields/#django.db.models.DateField.auto_now

Reply:
Good catch! I don't think using the `default` parameter will set `modified` upon modification, so changed those.